app/src/androidTest/java/org/mozilla/fenix/ui/TabbedBrowsingTest.kt

```diff
@@ -328,7 +326,8 @@ class TabbedBrowsingTest {
             verifyTabsTrayCounter()
             verifyExistingTabList()
             verifyNormalBrowsingNewTabButton()
-            verifyOpenedTabThumbnail()
+            // Disabled step due to ongoing tabs tray compose refactoring
```

Comment:
We should link a reference bug if we know any.

Reply:
Done !